tests/cases/examples/box_shadow_use.60

```diff
@@ -27,10 +27,11 @@ TestCase := Window {
 
         Rectangle {
             background: blue;
+            border-radius: 10px;
             drop-shadow-offset-x: 10px;
             drop-shadow-offset-y: 10px;
             drop-shadow-color: #00000080;
-            drop-shadow-blur: 5px;
+            drop-shadow-blur: 5;
```

Comment:
The updater test will fail.  Maybe the updater can wrap expression of `drop-shadow-blur` in a `(...) / 1px`

Reply:
Ah yes, well spotted! Good idea.